internal/app/singularity/action_flags.go

```diff
@@ -242,7 +242,7 @@ func initNamespaceVars() {
 	actionFlags.SetAnnotation("uts", "envkey", []string{"UTS", "UNSHARE_UTS"})
 
 	// -u|--userns
-	actionFlags.BoolVarP(&UserNamespace, "userns", "u", false, "run container in a new user namespace, allowing Singularity to run completely unprivileged on recent kernels. This may not support every feature of Singularity.")
+	actionFlags.BoolVarP(&UserNamespace, "userns", "u", false, "run container in a new user namespace, allowing Singularity to run completely unprivileged on recent kernels. This may not support every feature of Singularity. (Sandbox image only)")
```

Comment:
I think the new parenthetical addition is a bit confusing because the previous sentence is in the negative; it isn't completely clear if the parenthetical should also be considered in the negative; does it mean Sandbox image is the only thing *not* supported?  I suggest instead changing the last sentence to "This disables some features of Singularity, for example it only works with sandbox images."

Reply:
Sounds good to me. Want to make the PR @DrDaveD?

Reply:
#2592